hugo/layouts/partials/home/showcases.html

```diff
@@ -1,15 +1,45 @@
-<h1>Showcases</h1>
-{{ range $p := $.Site.Data.home.showcases.showcases }}
-    <div>
-        <h2>{{$p.title}}</h2>
-        <p>{{$p.desc}}</p>
-        <a href="{{ $p.link }}" target="_blank">check this project</a>
+<div class="showcases">
+    <div class="showcases__info">
+        <h1>Showcases</h1>
```

Comment:
For this `h1` you need an element that resets the headers, like `article` or `section`.